### ISSUE TITLE
Maybe downgrade swift version to 5.6

### ISSUE THREAD

**Issue by ruurd** [apple/swift-book#26]
Is it possible to downgrade to 5.6 to make it usable for mere mortals that run latest Monterey?

**Comment by amartini51** [comment]
The version of TSPL on Swift.org always shows the latest version of Swift that's available in Xcode or in an Xcode beta.  However, you can see both the Swift 5.6 and Swift 5.7 (beta) versions of TSPL on Apple Books:

<https://books.apple.com/us/book/the-swift-programming-language-swift-5-6/id881256329>

Under an older publication pipeline, we published two versions of TSPL on the web — beta and non-beta.  Maybe we sould turn this GitHub issue into a request to publish both versions again when we use DocC.

**Comment by rtharston** [comment]
I agree with amartini51, it would be nice to have both beta and release docs available.

@ruurd, I understand you are referring to not running a beta (Ventura or Xcode), but in case you weren't aware, you can run Xcode 14 on Monterey as well (and it is now out of beta, if that was holding you back). :)

**Comment by amartini51** [comment]
Discussed again on the forum: https://forums.swift.org/t/72375

To have it written down, we've had the following things available over time:

- When TSPL was hosted on developer.apple.com, there were separate web versions of the book for beta and prerelease, separate ePUB versions in the Apple Books store for beta and prerelease, and an ePUB version on Swift.org for only the latest.
- When TSPL was hosted on Swift.org using its pre-DocC toolchain, there was a single web version for the latest.  There were separate ePUB versions in the Apple Books store for beta and prerelease, and an ePUB version on Swift.org for only the latest. 
- Currently, with TSPL hosted on Swift.org and built using DocC, there is a single web version for the latest.  We're no longer able to build an ePUB version (see #2).
